src/views/login/login.js

```diff
@@ -0,0 +1,68 @@
+        <button class="credential-login-button credential-login-buttonT" type ="submit" value="Login">Login</button>
+    </div>
+</form>`
+}
+
+const SubmitFeedbackSection = function(option = {}){
+    return `
+    <div class= "grid" >
+    <div class="grid-image">
+        <img  src="/assets/new feedback.jpg" alt="new feedback" >
+    </div>
+    <article class="grid-text">
+        <h3 class="grid-text-padding">Submit new feedback</h3>
+        <p class="grid-text-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et tempus eros. Suspendisse scelerisque dapibus turpis, vel suscipit nisl pellentesque nec. Etiam pellentesque in enim volutpat euismod. Vivamus id nibh eget nibh egestas congue. Nunc ante tellus, ultrices vitae egestas vel, gravida quis tellus. Praesent eget magna justo. Ut ligula tortor, imperdiet vitae est et, porttitor ornare massa. Praesent semper gravida lacus molestie aliquet. Fusce mollis tristique massa non maximus. Proin varius nisl eget faucibus feugiat. Fusce laoreet accumsan purus id hendrerit. Morbi felis ante, dictum ac facilisis sed, maximus vitae felis. Integer imperdiet feugiat diam, in interdum tortor porttitor varius. Cras luctus, est eget tristique finibus, massa augue pulvinar quam, non placerat lectus erat nec ex. Nunc vestibulum dui nisi, id aliquam dui sollicitudin a. Vestibulum sit amet mi sed nisl eleifend fermentum ut vel diam. </p>
+    </article>
+    </div>
+    `
+}
+
+
+const SubmitFeedbackResult = function(option = {}){
+    return `
+    <div class="special-color"> 
+    <div class= "grid" >
+        <article class="grid-text">
+            <h3 class="grid-text-padding" >View feedback results</h3>
+            <p class="grid-text-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec et tempus eros. Suspendisse scelerisque dapibus turpis, vel suscipit nisl pellentesque nec. Etiam pellentesque in enim volutpat euismod. Vivamus id nibh eget nibh egestas congue. Nunc ante tellus, ultrices vitae egestas vel, gravida quis tellus. Praesent eget magna justo. Ut ligula tortor, imperdiet vitae est et, porttitor ornare massa. Praesent semper gravida lacus molestie aliquet. Fusce mollis tristique massa non maximus. Proin varius nisl eget faucibus feugiat. Fusce laoreet accumsan purus id hendrerit. Morbi felis ante, dictum ac facilisis sed, maximus vitae felis. Integer imperdiet feugiat diam, in interdum tortor porttitor varius. Cras luctus, est eget tristique finibus, massa augue pulvinar quam, non placerat lectus erat nec ex. Nunc vestibulum dui nisi, id aliquam dui sollicitudin a. Vestibulum sit amet mi sed nisl eleifend fermentum ut vel diam. </p>
+        </article>
+        <div class="grid-image">
+            <img  src= "/assets/results.jpg" alt="results">	
+        </div>
+</div>
+    `
+}
+const Footer = function(option= {}){
+    return `<footer class="footer footerT" >
```

Comment:
I'd go ahead and expand this `footerT` class to something more meaningful, like `footer-theme`. It's easier for anyone looking at the code to understand what's happening here, regardless of whether or not they're familiar with the context.